src/Test.FunctionalTests.Sarif/Multitool/BaselineOptionTests.cs

```diff
@@ -87,18 +91,18 @@ protected override string ConstructTestOutputFromInputResource(string inputResou
             string testName = parameter as string;
 
             string fileToBeValidated = "ToBeValidated.sarif";
-            string filePathToBeValidated = Path.Combine(this.OutputFolderPath, fileToBeValidated);
-            string logText = GetResourceText($"Inputs.{fileToBeValidated}");
```

Comment:
>GetResourceText [](http://example.com/codeflow?start=29&length=15)

Previously, we would explicitly retrieve a resource knowing that it lived in disk in a directory named 'Inputs' by qualifying a file name with 'Inputs.'. This matches the resource manifest path created from the directory name. 

Now we ask explicitly for an 'input' file (there is also special knowledge of the 'expected output' files. This will drive us to consistency across tests, i.e., storing files in consistent directory names.

Note that by redacting 'Inputs.' from the file name passed to the actual SDK API, we change resulting URLs for the fake file names in checked in SARIF content. This results in many changes in the TestData directory, later.